lib/DxcSupport/WinFunctions.cpp

```diff
@@ -51,7 +51,19 @@ HRESULT IntToUInt(int in, UINT *out) {
     hr = S_OK;
   } else {
     *out = 0xffffffff;
-    hr = (HRESULT)1L;
+    hr = ERROR_ARITHMETIC_OVERFLOW;
```

Comment:
Should  `UIntAdd` above not return this error as well?

Reply:
ah yes. missed that.